src/database/DatabaseManager.js

```diff
@@ -80,41 +88,93 @@ export class DataBaseManager {
+        // Get the discussion related to the id in parameter
         return Discussion.findOne({_id: anIdDiscussion}, function (err, discussion) {
             if (err || discussion == null) {
                 logger.debug(`Error when requesting discussion`);
+            }
+            else {
+                logger.debug(discussion);
+            }
+        });
+    }
+
+    /**
+     * Get a question proposed by a user in the database with the corresponding id
+     * @param anIdQuestion integer that represents the id of the question desired
+     * @param anIdDiscussion integer that represents the id of the discussion related to the question
+     * @returns {Promise<Query|void>} A Question or null if not found
+     */
+    async getUserQuestion(anIdQuestion, anIdDiscussion){
+        logger.debug(`Getting the Question with id ${anIdQuestion}`);
+        // Get the discussions related to the id
+        return QuestionSuggestion.findOne({"id.refQuestion": anIdQuestion, "id.refDiscussion": anIdDiscussion}, function (err, question) {
+            if (err || question == null) {
+                logger.debug(`Error when requesting discussion`);
             } else {
-                console.log(discussion);
+                logger.debug(question);
             }
         });
     }
 
     /**
-     * Get the discussions of an administrator
-     * @param username String that is the username of the administrator
-     * @returns a Array of Discussion that represents the discussions started by an user
+     * Get all the questions proposed by users and accepted by admin during a discussion
+     * @param anIdDiscussion integer that represents the id of the discussion related to questions that we want
+     * @returns {Promise<*>} an array of Questions or undefined if no questions with the id of the discussion passed are found
      */
-    async getDiscussionsAdmin(username){
+    async getAcceptedQuestionsSuggestion(anIdDiscussion){
+        logger.debug(`Getting the Question in the debate ${anIdDiscussion}`);
+        // Get the discussions related to the id
+        return QuestionSuggestion.find({"id.refDiscussion": anIdDiscussion, approved: true}, function (err, questions) {
+            if (err || questions == null) {
+                logger.debug(`Error when requesting question : No questions were found`);
+            } else {
+                logger.debug(questions);
+            }
+        });
+    }
+
+    /**
+     * Get all the questions proposed by users but that are not yet accepted
+     * @param anIdDiscussion integer that represents the id of the discussion related to questions that we want
+     * @returns {Promise<*>} an array of Questions or undefined if no questions with the id of the discussion passed are found
+     */
+    async getNotYetAcceptedQuestionsSuggestion(anIdDiscussion){
+        logger.debug(`Getting the Question in the debate ${anIdDiscussion}`);
```

Comment:
Same here. It would also be interesting to log the type of question (NotYetAcceptedSugestion?).

Reply:
I did not find a better name for this function because the database will only contain the approved question and the question that has not been yet approved.
The unapproved question will be directly deleted.
Does it make sense?

Reply:
I've no problem with the function name. I was asking for the text with `logger.debug`. Would it be better to have this
```
logger.debug(`Getting the Question in the debate ${anIdDiscussion}`);
```
or something like 
```
logger.debug(`Getting the not yet approved Question in the debate ${anIdDiscussion}`);
```

Reply:
Ha ok if it is that I have changed it

Reply:
Noice